src/rpc/rpc.cc

```diff
@@ -1198,6 +1199,10 @@ future<> server::connection::send_unknown_verb_reply(std::optional<rpc_clock_typ
       return isolation_config{};
   }
 
+  void server::warn_stuck_handler() {
+      seastar_logger.warn("Server cannot stop for too long, {} handlers are running", _handlers_running);
```

Comment:
why not simply print `_reply_gate.get_count()`?

Reply:
Because reply gate also covers code that's trying to send message over the network. The intention is to _just_ guard the callback execution